docs/source/usage/01_usage.md

```diff
@@ -99,15 +99,15 @@ If you want to list multiple hooks, you have to use the following parameter name
 - `post_package_build`: after building packages against sources
 - `pre_container_build`: before building container images
 - `post_container_build`: after building container images
+- `pre_operator_build`: before building operators
+- `post_operator_build`: after building operators
 - `pre_deploy`: before deploying EDPM
+- `post_ctlplane_deploy`: after Control Plane deployment (not architecture)
 - `post_deploy`: after deploying EDPM
-- `post_ctlplane_deploy`: after Control Plane deployment
-- `pre_tests`: before running tests
-- `post_tests`: after running tests
 - `pre_admin_setup`: before admin setup
 - `post_admin_setup`: before admin setup
-- `pre_reporting`: before running reporting
-- `post_reporting`: after running reporting
```

Comment:
the `post_reporting` is also used in the example below, line 115. You may want to change that one with an existing hook?

Reply:
Right, fixed, thanks